framework/filters/Cors.php

```diff
@@ -80,6 +80,10 @@ class Cors extends ActionFilter
      */
     public $actions = [];
     /**
+     * @var bool handle the preflight check and don't proceed with the execution of the request.
+     */
+    public $handleOptions = true;
```

Comment:
OK. It makes sense. I don't like the name of the property though. "Handle" is too general. I'd name it like "skipOptions", "allowOptions" or alike. Do you have any naming ideas?